core/workspace_drag_surface_svg.js

```diff
@@ -117,15 +117,17 @@ WorkspaceDragSurfaceSvg.prototype.getSurfaceTranslation = function() {
  */
 WorkspaceDragSurfaceSvg.prototype.clearAndHide = function(newSurface) {
   if (!newSurface) {
-    throw Error('Couldn\'t clear and hide the drag surface: missing ' +
+    throw Error(
+        'Couldn\'t clear and hide the drag surface: missing ' +
         'new surface.');
```

Comment:
Nit: I _think_ this will fit on a single line now.

Reply:
Done!